cmd/mirror-main.go

```diff
@@ -516,8 +516,9 @@ func (mj *mirrorJob) startMirror(ctx context.Context, cancelMirror context.Cance
 				return
 			}
 			if sURLs.Error != nil {
+				stopParallel()
 				errCh <- sURLs.Error
-				continue
+				return
```

Comment:
A small comment @vadmeste. I can see that this will return on the first failure itself. For eg, If there is a broken symlink in the source and if the listing reports it,  we should not return right away right, Should'nt we ignore the faulty file and continue mirroring the rest?

Reply:
@Praveenrajmani listing doesn't return broken symlink anymore, this is fixed recently. Otherwise I think it is okay to just quit, I don't see any use case why we should continue otherwise.

Reply:
:+1: